unit_tests/test_views.py

```diff
@@ -188,13 +189,19 @@ def test_essays(self):
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, u'Rëd Sox')
 
+    @responses.activate
     def test_specific_essay(self):
-        models.Essay.objects.create(slug='ger', author='David Ortiz', title=u'Rëd Sox', text='### Red Sox lineup[^n1]\n\n[^n1]: footnote text')
+        responses.add(responses.GET, 'https://localhost/api/search',
+        body=json.dumps({'response': {'docs': [{'pid': 'testsuite:230605'}]}}),
+        status=200,
+        content_type='application/json'
+        )
```

Comment:
Here's the error from Travis: 
Connection refused: GET https://localhost/api/search/?q=ir_collection_id:621+AND+display:BDR_PUBLIC+AND+(pid:%22testsuite:230605%22)&fl=primary_title,rel_has_pagination_ssim,rel_is_part_of_ssim,creator,pid,genre

Maybe you need a slash on the end of the url (ie. https://localhost/api/search/). But you can try it on your laptop and make sure it works: "python run_unit_tests.py"

Reply:
Going to try pushing again (sorry) because I'm getting a weird error running tests on my local machine:
~~~~
FAIL: test_init (unit_tests.test_models.TestBDRObject)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/Zak/Documents/rome_project/ttwr/unit_tests/test_models.py", line 37, in test_init
    self.assertFalse(b)
AssertionError: <rome_app.models.BDRObject instance at 0x107726b00> is not false
~~~~
Let's see if the slash fixed things.